app/views/membership.html

```diff
@@ -203,35 +208,38 @@
                           </ui-select-choices>
                         </ui-select>
                       </div>
+
+                      <span
+                        ng-if="newBinding.kind === 'ServiceAccount'"
+                        class="mar-left-md mar-right-md hidden-xs">/</span>
+
+                      <!-- name entry/picker for service accounts -->
+                      <div
+                        ng-if="newBinding.kind === 'ServiceAccount'"
+                        class="service-account-name">
+                        <ui-select
+                          ng-model="newBinding.name"
+                          theme="bootstrap"
+                          search-enabled="true"
+                          title="Select a service account"
+                          class="select-role">
+                          <ui-select-match
+                            placeholder="Service account">
+                            <span ng-bind="newBinding.name"></span>
```

Comment:
I think I have that because it was part of `ui-select` docs when i first dropped them in, I'll swap to the `{{}}`.